app/helpers/token.js

```diff
@@ -1,16 +1,18 @@
 const moment = require('moment');
 const jwt = require('jsonwebtoken');
-const { secret, expiration } = require('../../config').common.session;
+const { secret, expiration, type } = require('../../config').common.session;
 
 exports.createToken = any => {
+  const exp = moment().add(expiration, type);
   const payload = {
     sub: any,
     iat: moment().unix(),
-    exp: moment()
-      .add(expiration, 'days')
-      .unix()
+    exp: exp.unix()
+  };
+  return {
+    token: jwt.sign(payload, secret),
+    expiration: exp.toDate()
```

Comment:
why do you return this field?

Reply:
to return the expiration date of the token as the first acceptance criteria of the current card says